wiki/Tournaments/GTS/BGTS_2020/en.md

```diff
@@ -0,0 +1,216 @@
+---
+tags:
+  - BGTS2020
+  - BGTS
+  - GTS
+---
+
+# Beginners Global Taiko Showdown
+
+![BGTS 2020 logo](img/logo.jpg)
+
+The **Beginners Global Taiko Showdown 2020** (***BGTS 2020***) is a Beginner worldwide country-based double-elimination 2v2 osu!taiko tournament hosted by ![][flag_FR] [Kasumii-sama](https://osu.ppy.sh/users/6177263), ![][flag_FR] [TLQ\_Yoshii](https://osu.ppy.sh/users/7157133), and ![][flag_FR] [Un bretzel](https://osu.ppy.sh/users/1542565). It is the first installment of the Beginner Global Taiko Showdown and part of the Global Taiko Showdown series.
```

Comment:
"beginner" is kinda vague, I think you should remove that and state the rank restriction here instead